src/utils/mod.rs

```diff
@@ -5,6 +5,7 @@ pub mod args;
 pub mod auth_token;
 pub mod chunks;
 pub mod cordova;
+pub mod derive_parser;
```

Comment:
Regarding the naming/location of this module: to clarify, this is going to be shared between all commands using the new derive API, i.e. it is not specific to metrics?

I would also suggest that we consider moving this to somewhere inside `src/commands` since the logic is related to parsing commands, rather than being a general utility.